src/api/index.ts

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export const axiosInstance = axios.create({
+  baseURL: 'https://react-gift-mock-api-sigma.vercel.app/',
+  timeout: 10000,
```

Comment:
추가로 큰 숫자를 사용할 때는 numeric sperator를 같이 사용해보시면 좋을거 같습니다 ㅎㅎ
```suggestion
  timeout: 10_000,
```